cpp/src/Ice/OutgoingResponse.cpp

```diff
@@ -136,10 +131,6 @@ namespace
             replyStatus = ReplyStatus::UnknownLocalException;
             ostringstream str;
             str << ex; // this includes more details than ex.what()
-            if (IceInternal::printStackTraces)
```

Comment:
We still need to decide what we include in the "reason" message we transmit over the wire. I believe we currently transmit too much. That's for another PR in C++ and other languages.